server/views/pages/componentRequestForm.njk

```diff
@@ -5,238 +5,278 @@
 {% from "govuk/components/select/macro.njk" import govukSelect %}
 {% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %} 
 {% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
 
 {% block content %}
-  <div class="govuk-grid-row govuk-body">
+<div class="govuk-grid-row govuk-body">
     {% include 'partials/errorSummary.njk' %}
-<h1 class="govuk-heading-xl">Component Request Form</h1>
+    <h1 class="govuk-heading-xl">Component Request Form</h1>
 
-<h2>After submitting this form, SRE team will run the bootstrap job to create the github repo and service catalog entries. </h2>
+    <h2>After submitting this form, SRE team will run the bootstrap job to create the github repo and service catalog entries. </h2>
+    <div class="govuk-grid-column-one-half">
+    <form action="/forms/component-request-form" method="POST">
+      {%- set teams = [] -%}
+      {%- set products = [] -%}
+      {%- for product in productList -%}
+      {%- set products = (products.push({
+        value: product.value,
+        text: product.text,
+        selected: product.value == submittedForm.product
+      }), products) -%}
+      {%- endfor -%}
+      {%- for team in teamList -%}
+        {%- set teams = (teams.push({
+          value: team.value,
+          text: team.text,
+          selected: team.value == submittedForm.requester_team
+        }), teams) -%}
+      {%- endfor -%}
+      <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+      {{ govukInput({
+        id: "github_repo",
+        name: "github_repo",
+        value: submittedForm.github_repo,
+        classes: "govuk-!-width-one-half",
+        label: {
+          text: "Github repository name *",
+          classes: "govuk-fieldset__legend--s"
+        },
+        hint: {
+          text: "must start with hmpps- , contains only alphanumeric characters, only special character allowed Hyphen(-)"
+        },
+        errorMessage: validationErrors | findError('github_repo')
+      }) }}
+        {{ govukTextarea({
+        id: "repo_description",
+        name: "repo_description",
+        value: submittedForm.repo_description,
+        classes: "govuk-!-width-two-third",
+        label: {
+          text: "Github Repository Description *",
+          classes: "govuk-fieldset__legend--s"
+        },
+        errorMessage: validationErrors | findError('repo_description')
+      }) }}
+      {{ govukRadios({
+        classes: "govuk-radios--inline",
+        idPrefix: "base_template",
+        name: "base_template",
+        value: submittedForm.base_template,
+        errorMessage: validationErrors | findError('base_template'),
+        fieldset: {
+          legend: {
+            text: "Github template repository *",
```

Comment:
we probably should support creating repos that aren't based on a template also.. maybe as a follow up?

Reply:
Added github-only option